services/connectivity/src/service/connectivity.js

```diff
@@ -125,29 +118,72 @@ module.exports = (connectivityPersistence) => {
      * connectivity
      */
     getCurrentPAConnectivityBySE: async (areaType, areaId, seType) => {
-      let data;
+      let rawData;
+      let paConnDataInSE;
+      let totalArea;
       switch (seType) {
         case 'Páramo':
-          data = connectivityPACurrentParamo;
-          break;
+          rawData = await connectivityPersistence.findCurrentPAConnectivityInParamo(
+            areaTypeKeys(areaType), areaId,
+          );
+          paConnDataInSE = rawData[0] ? rawData[0] : null;
+          if (!paConnDataInSE) {
+            throw new Error(
+              'Data for Current PA Connectivity In Paramo doesn\'t exists in the selected area id and area type',
+            );
+          }
+          totalArea = Number(paConnDataInSE.area_ha);
+          delete paConnDataInSE.area_ha;
+          return Object.keys(paConnDataInSE).map(key => (
+            {
+              key: paConnCategoriesKeys(key),
+              area: Number(totalArea) * Number(paConnDataInSE[key]) / 100,
+              percentage: Number(paConnDataInSE[key]),
+            }
+          ));
```

Comment:
These lines of code are repeated many times, it would be better to define `rawData` inside the switch-case and let this section of code only once

Reply:
Done